### PR TITLE
feat: #2116 node connect session

### PR DESCRIPTION
**Pull request by willmcvay**
# Pull request checklist

## Does this close any currently open issues?

fixes: #2116 

**Please check if your PR fulfills the following requirements:**

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Build (`yarn build`) was run locally and any changes were pushed
- [x] Lint (`yarn lint`) has passed locally and any fixes were made for failures
- [x] Test (`yarn test`) has passed locally and any fixes were made for failures

## Pull request type

**Please check the type of change your PR introduces:**

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: #2116 

## What is the new behavior?

- Adds node version of connect session
- Refactors AML and CRA template so singleton does not get a runtime issue

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: packages/aml-checklist/src/core/index.tsx [hunk 1]
```diff
@@ -36,27 +35,34 @@ export const renderApp = (Component: React.ComponentType) => {
...
+
+    window.reapit.config = config
+
+    // I import the app dynamically so that the config is set on window and I avoid any
+    // runtime issues where config is undefined
+    const { default: App } = await import('./app')
+
+    renderApp(App)
+  } catch (error) {
+    console.error('Cannot fetch config', error)
```
Comment (phmngocnghia) [inline packages/aml-checklist/src/core/index.tsx:64]: LGTM. This error seems geneirc. At this error in this big try catch could belong to other reasons that not involve fetching `config.json`.  E.g.: ReactGA is failed to initialize and throw

**Comment by phmngocnghia** [comment]
Could you add connection-session as dep for aml-checklist. It's used by lerna to rerun aml pipeline when connect-session code change.

**Comment by phmngocnghia** [comment]
@willmcvay I created a new bug about AML auth: https://github.com/reapit/foundations/issues/2161. You may want to check it

File: packages/aml-checklist/src/core/index.tsx (hunk 1 above)
Comment (willmcvay) [reply packages/aml-checklist/src/core/index.tsx:64]: TBH, it's just the error that was there before, I'll have a quick look

**Comment by willmcvay** [comment]
> @willmcvay I created a new bug about AML auth: #2161. You may want to check it

Have closed, see comments on the ticket, this was intentional

**Comment by willmcvay** [comment]
> Could you add connection-session as dep for aml-checklist. It's used by lerna to rerun aml pipeline when connect-session code change.

I can now, looks like has just been deployed

**Pull request merged**